love/src/components/ScriptQueue/ConfigPanel/ConfigPanel.jsx

```diff
@@ -861,7 +910,7 @@ export default class ConfigPanel extends Component {
                 title="Enqueue script"
                 size="large"
                 onClick={this.onLaunch}
-                disabled={[ERROR, VALIDATING, NEED_REVALIDATION].includes(this.state.validationStatus)}
+                disabled={[ERROR, VALIDATING, NEED_REVALIDATION, EMPTY].includes(this.state.validationStatus)}
```

Comment:
No Sebastian, for example when yamlSchema is a comment, yaml.load return a null, and yamlData in this case it's difference to {}, and fail component.

Reply:
ok, I see. Thanks for claryfing that :raised_hands:! This will be the case for scripts which doesn't have a schema right? In that case dismiss this comment please.